doc/user_guide/user_guide.md

```diff
@@ -57,20 +57,40 @@ models on the Exasol Cluster. More information on The BucketFS can be found
 - BucketFS Connection 
   - An Exasol connection object must be created with Exasol BucketFS connection 
   information and credentials. 
-  - An example connection object is created as follows: 
+  - A format of the connection object is as following: 
   ```sql
   CREATE OR REPLACE CONNECTION <BUCKETFS_CONNECTION_NAME>
       TO '<BUCKETFS_ADDRESS>'
       USER '<BUCKETFS_USER>'
       IDENTIFIED BY '<BUCKETFS_PASSWORD>'
   ```
-
-  - The `BUCKETFS_ADDRESS` looks like the following:
-  
-    **Note:** The `<PATH_IN_BUCKET>` can not be empty.
-  ```buildoutcfg
-    http[s]://<BUCKETFS_HOST>:<BUCKETFS_PORT>/<BUCKET_NAME>/<PATH_IN_BUCKET>;<BUCKETFS_NAME>
+'<BUCKETFS_ADDRESS>', '<BUCKETFS_USER>' and '<BUCKETFS_PASSWORD>' are JSON strings whose content depends on the storage backend.
+Below is the description of the parameters that need to be passed for On-Prem and SaaS databases. The distribution of
+the parameters among those three JSON strings do not matter.
+
+**On-Prem Database**
+* url: Url of the BucketFS service, e.g. "http(s)://127.0.0.1:2580".
+* username: BucketFS username (generally, different from the DB username).
+* password: BucketFS user password.
+* bucket_name: Name of the bucket in the BucketFS.
+* verify: Optional parameter that can be either a boolean, in which case it controls whether the server's
+    TLS certificate is verified, or a string, in which case it must be a path to a CA bundle to use. Defaults to ``true``.
```

Comment:
I think, we should mention that the path needs to be accessible from inside the UDF

Reply:
Oh, yes, it's a good point, I forgot about this. In the notebook-connector we refuse to take a custom CA bundle, turning the verification off instead, as described [here](https://github.com/exasol/notebook-connector/blob/a3cf77a694b54d09c9be22de6525d283bb4022da/exasol/nb_connector/extension_wrapper_common.py#L130). Shall we take a unified approach? What do you think we should do?